### DISCUSSION TITLE
How to use cli command?

### DISCUSSION THREAD

**Discussion by ucxl** [LSPosed/LSPosed#1949] [Q&A] [answered]
How to use cli command? Is there a tutorial？ tks！

**Comment by JasonKhew96** [comment, marked as answer]
This feature doesn't exist
https://github.com/LSPosed/LSPosed/issues/405

**Reply by mywalkb** [reply]
it will never exist. After more than a year it is still in TODO.
In my mod mywalkb/LSPosed_mod CLI exist if anyone is interested. @ucxl already tried it
